### DISCUSSION TITLE
POST Rest API issue

### DISCUSSION THREAD

**Discussion by Abd-Atif** [Budibase/budibase#9990] [Help] [answered]
Hi Folks,
Anyone here can help me? 
I want to use the value that I got from the Fields of Form in the design as a body of a REST API POST request. Here is how I write it in REST API datasource.
Also Im getting this error:
![image](https://user-images.githubusercontent.com/94058099/224531575-c100a837-02fc-4fde-8933-84fe77baad02.png)
here are screenshots for better understanding:
![image](https://user-images.githubusercontent.com/94058099/224531564-3770459f-fcb9-45b9-afaf-0d963d4fc301.png)

![image](https://user-images.githubusercontent.com/94058099/224531566-1c14c561-cd0f-49ce-984f-778d457842f0.png)

thanks

**Comment by melohagan** [comment, marked as answer]
Hey @Abd-Atif 

You are missing a comma on line 8 (should be after the Binding x).

If that doesn't resolve it, you could try wrapping the bindings within quotes, e.g. `"{{ Binding.a }}"`

**Reply by Abd-Atif** [reply]
Thanks @melohagan  :)
